app/main.py

```diff
@@ -0,0 +1,118 @@
+import hashlib
+import json
+from functools import lru_cache
+from pathlib import Path
+from uuid import UUID
+
+import uvicorn
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from pydantic.json_schema import SkipJsonSchema
+
+SCHEMAS_PATH = Path(Path(__file__).parent.parent, "schemas")
+
+app = FastAPI()
+
+
+class CiMetadata(BaseModel):
+    """
+    Model for collection instrument metadata
+    copied from https://github.com/ONSdigital/eq-cir-fastapi/blob/main/app/models/responses.py#L21
+    """
+
+    # Required fields
+    ci_version: int
+    data_version: str
+    form_type: str
+    id: UUID
+    language: str
+    published_at: str
+    schema_version: str
+    status: str
+    survey_id: str
+    title: str
+    description: str
+    # Optional fields
+    sds_schema: str | SkipJsonSchema[None] = ""
+
+
+def generate_guid_for_schema(schema_name: str, schema_language: str) -> UUID:
+    """schema_name and language together are unique so are suitable for generating a guid"""
+    combined_hash = hashlib.sha256(
+        f"{schema_language}_{schema_name}".encode("utf-8")
+    ).hexdigest()
+    return UUID(combined_hash[:32])
+
+
+def get_ci_metadata(guid: UUID, schema_name: str, schema_json: dict) -> CiMetadata:
+    """
+    For any fields where runner test schemas may be missing data that is mandatory in the CIR model,
+    use a mock value for a more realistic response.
+    """
+    mock_data = {
+        "published_at": "2021-01-01T00:00:00.0000000Z",
+        "description": "Mock description",
```

Comment:
Worth setting this from the schema.
```suggestion
        "description": schema_json.get("description", "Mock description"),
```

Reply:
It's currently doing this already, since I've got `**mock_data, **schema_json` so the schema ones overwrite any mock ones if present

Reply:
Ah right yes, 🙈 👍